wgpu-core/src/command/bundle.rs

```diff
@@ -745,7 +745,7 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
                         num_dynamic_offsets,
                         bind_group_id,
                     } => {
-                        let scope = PassErrorScope::SetBindGroup;
+                        let scope = PassErrorScope::SetBindGroup(bind_group_id);
```

Comment:
if we go this route, wouldn't it be easier to just have the full `command` cloned into the error?

Reply:
Maybe. I don’t currently see a reason though, what else useful information would be had, that isn’t then also displayed in the more specific error?

I don’t see the implementation to be any simpler the either.

Reply:
The main benefit is not having a separate `PassErrorScope` enum at all. If you don't think that's a big one, I'm fine with proceeding with your plan :)